solutions/instances/main.tf

```diff
@@ -348,7 +359,7 @@ resource "ibm_iam_authorization_policy" "policy" {
   target_service_name         = local.kms_service
   target_resource_instance_id = local.existing_kms_guid
   roles                       = ["Reader"]
-  description                 = "Allow the COS instance with GUID ${local.cos_instance_guid} reader access to the kms_service instance GUID ${local.existing_kms_guid}"
+  description                 = local.existing_kms_guid != null ? "Allow the COS instance with GUID ${local.cos_instance_guid} reader access to the kms_service instance GUID ${local.existing_kms_guid}" : null
```

Comment:
I don't think you need this ternary logic here because if the count is correct, it won't enter this resource block unless `existing_kms_guid` has a value

Reply:
I hit the error twice. Let me rechk on this.

Reply:
If you got an error then then there is a bug somewhere in the count logic